package.json

```diff
@@ -73,6 +73,8 @@
     "naf-janus-adapter": "^2.0.2",
     "networked-aframe": "github:mozillareality/networked-aframe#master",
     "nipplejs": "github:mozillareality/nipplejs#mr-social-client/master",
+    "node-ensure": "0.0.0",
```

Comment:
Whats with this dependency? Dont see it being imported and doesn't appear to be a peer dependency of pdfjs

Reply:
the build was failing complaining about it due to `pdfjs-dist` needing it. i've seen this happen in the past with other libraries, not sure